modules/oe_webtools_etrans/tests/src/Kernel/ETransBlockTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_etrans\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\language\Entity\ConfigurableLanguage;
+use Drupal\oe_webtools_etrans\Plugin\Block\ETransBlock;
+
+/**
+ * Tests the block that displays the Webtools eTrans link.
+ */
+class ETransBlockTest extends KernelTestBase {
```

Comment:
I think checking the rendered script tag, as is done here, is enough. Webtools should provide the functional testing.

Reply:
I agree, this is the limit of our responsibility. This is also consistent with the other modules in `oe_webtools`.